tests/Integration/CategoryFrontendControllerTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace IntegerNet\GlobalCustomLayout\Test\Integration;
+
+use IntegerNet\GlobalCustomLayout\Test\Util\CategoryLayoutUpdateManager;
+use Magento\Catalog\Api\CategoryRepositoryInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+
+/**
+ * Tests whether global layout handles are correctly saved on categories
+ * and retrieved on the frontend on Category views
+ */
+class CategoryFrontendControllerTest extends AbstractFrontendControllerTest
+{
+    /**
+     * Check that Global Custom Layout Update files work for Category views.
+     *
+     * @return void
+     * @throws \Magento\Framework\Exception\CouldNotSaveException
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
+     *
+     * @magentoDataFixture Magento/CatalogUrlRewrite/_files/categories_with_product_ids.php
+     */
+    public function testViewWithGlobalCustomUpdate(): void
+    {
+        //Setting a fake file for the category.
+        $file = 'test-file';
+        $categoryId = 5;
+
+        /** @var CategoryLayoutUpdateManager $layoutManager */
+        $layoutManager = Bootstrap::getObjectManager()->get(CategoryLayoutUpdateManager::class);
```

Comment:
you could use `$this->objectManager` to shorten the code a bit, since you are instantiating it in the abstract setup